LDKNodeMonday/View/Home/ImportWalletView.swift

```diff
@@ -0,0 +1,98 @@
+//
+//  ImportWalletView.swift
+//  LDKNodeMonday
+//
+//  Created by Daniel Nordh on 28/11/2024.
+//
+
+import BitcoinUI
+import LDKNode
+import SwiftUI
+
+struct ImportWalletView: View {
+    @Environment(\.dismiss) private var dismiss
+
+    @AppStorage("isFirstTime") var isFirstTime: Bool = true
```

Comment:
requested change: could we re-use `isOnboarding` instead of adding `isFirstTime`? It would be nice to not add `isFirstTime` if we dont have to, and in my head I equate what I'm doing with `isOnboarding` (persons first time == they are onboarding) so I think you can just use `isOnboarding`? but I might be totally missing something and it not working for you if you did that.

so that would let us not have to add `isFirstTime` in 2 views:
-`ImportWalletView`
-`OnboardingView`

Reply:
Got it. This was in the original code but I never understood the difference between isOnboarding and isFirstTime.
Removed.

Reply:
thanks, resolving conversation too